cmd/restic/delete.go

```diff
@@ -12,7 +12,7 @@ func DeleteFiles(gopts GlobalOptions, repo restic.Repository, fileList restic.ID
 	deleteFiles(gopts, true, repo, fileList, fileType)
 }
 
-// DeleteFiles deletes the given fileList of fileType in parallel
+// DeleteFilesChecked deletes the given fileList of fileType in parallel
```

Comment:
Please rebase to the current master branch, the has already been fixed on master.